cypress/utils/utils.ts

```diff
@@ -789,7 +789,7 @@ export function performRowAction(itemName: string, action: string): void {
 // Perform edit/delete action on the specified row selector by clicking an icon button
 /* As of Tackle 2.1, this function can be used to click the update button and kebab menu for
    applications on the Application Inventory page */
-export function performRowActionByIcon(itemName: string, action: string): void {
+export function performRowActionByIcon(itemName: string, action: string, nth_position = 0): void {
```

Comment:
Nitpick to maintain consistency

```suggestion
export function performRowActionByIcon(itemName: string, action: string, nthPosition = 0): void {
```